tests/distributions/test_distributions_default.py

```diff
@@ -0,0 +1,59 @@
+from plone import api
+from Products.CMFCore.WorkflowCore import WorkflowException
+from zope.component.hooks import setSite
+
+import pytest
+
+
+@pytest.fixture()
+def portal(portal_default):
+    setSite(portal_default)
+    return portal_default
+
+
+class TestDistributionDefault:
+    @pytest.mark.parametrize(
+        "attr,expected",
+        [
+            ("title", "Plone Site"),
+            ("description", "A Plone Site"),
+            ("exclude_from_nav", False),
+        ],
+    )
+    def test_plone_site_attributes(self, portal, attr, expected):
+        assert getattr(portal, attr) == expected
+
+    @pytest.mark.parametrize(
+        "package,expected",
+        [
+            ("plone.restapi", True),
+            ("plone.app.contenttypes", True),
+            ("plone.app.caching", True),
+            ("plonetheme.barceloneta", True),
```

Comment:
barceloneta is installed when installing volto? 🤔

Reply:
Yes, it is. Otherwise the ClassicUI is totally broken